sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -100,6 +100,7 @@ export interface CreateBatchOptions extends OperationOptionsBase {
 export interface CreateQueueOptions extends OperationOptions {
     authorizationRules?: AuthorizationRule[];
     autoDeleteOnIdle?: string;
+    availabilityStatus?: EntityAvailabilityStatus;
```

Comment:
Can you add an entry in the changelog.md file?

Reply:
This is done.